install_config/install/docker_registry.adoc

```diff
@@ -107,11 +107,11 @@ $ oc volume deploymentconfigs/docker-registry \
      --source='{"nfs": { "server": "<fqdn>", "path": "/path/to/export"}}'
 ----
 
-There are link:#known-issues[known issues] when using multiple registry replicas
-with the same NFS volume. We recommend changing the docker-registry service's
-`sessionAffinity` to `ClientAPI` like this:
-
 [[session-affinity-workaround]]
+There are link:#registry-known-issues[known issues] when using multiple registry
+replicas with the same NFS volume. We recommend changing the docker-registry
+service's `*sessionAffinity*` to *ClientIP*:
```

Comment:
@miminar Can you ack the above `s/ClientAPI/ClientIP`? Seems it was just a typo that got introduced along the way.